docs/setup.asciidoc

```diff
@@ -56,6 +56,8 @@ public class Startup
 }
 ----
 
+NOTE: Make sure the `app.UseAllElasticApm(...)` line is the first line in the `Configure` method, otherwise the agent won't be able to properly measure the timing of your requests and potentially complete requests may be missed by the agent. 
```

Comment:
```suggestion
The `app.UseAllElasticApm(...)` line must be the first line in the `Configure` method, otherwise the agent won't be able to properly measure the timing of your requests, and potentially complete requests may be missed by the agent. 
```